xen/arch/arm/coproc/schedule.c

```diff
@@ -253,15 +253,20 @@ static s_time_t vcoproc_schedule_context_switch(struct vcoproc_instance *curr,
 
         wait_time = vcoproc_context_switch_from(curr);
 
-        if ( wait_time == 0 )
+        if ( likely(wait_time == 0) )
```

Comment:
As I understand (wait_time>0) is another `if` check, so it should be `likely()` as well keeping in mind that normally we do not expect `wait_time < 0`